docs/src/index.md

```diff
@@ -19,13 +19,30 @@
     For the documentation of JuMP 0.18, see
     [here](https://jump.dev/JuMP.jl/0.18/).
 
+## What is JuMP?
+
 [JuMP](https://github.com/jump-dev/JuMP.jl) is a domain-specific modeling
-language for [mathematical
-optimization](https://en.wikipedia.org/wiki/Mathematical_optimization) embedded
-in [Julia](https://julialang.org/). It currently supports a number of open-source
-and commercial solvers (see below) for a variety of problem classes, including
-**linear programming**, **mixed-integer programming**, **second-order conic
-programming**, **semidefinite programming**, and **nonlinear programming**.
+language for [mathematical optimization](https://en.wikipedia.org/wiki/Mathematical_optimization)
+embedded in [Julia](https://julialang.org/). It currently supports a number of
+open-source and commercial solvers for a variety of problem classes, including
+linear, mixed-integer, second-order conic, semidefinite, and nonlinear
+programming.
+
+## Resources for getting started
+
+* Checkout the [Installation Guide](@ref).
```

Comment:
Perhaps a TODO: I think it would be useful to have a page that explains the concept of an algebraic modeling language and what JuMP does do and doesn't do. The documentation is missing this kind of summary that would help people decide if they want to use JuMP or not. Jumping straight to the installation guide is making an assumption that somewhat has already decided to install JuMP, but maybe they're browsing the documentation to decide *if* they want to install JuMP.

Reply:
Opened an issue https://github.com/jump-dev/JuMP.jl/issues/2412